src/micro_ros_arduino.h

```diff
@@ -48,7 +48,14 @@ static inline void set_microros_transports(){
 #include <NativeEthernet.h>
 #endif
 
-#if defined(TARGET_STM32F4) || defined(ARDUINO_TEENSY41)
+#if defined(TARGET_PORTENTA_H7_M7) 
+#include <EthernetUdp.h>
+#include <PortentaEthernet.h>
+#include <uxr/client/transport.h>
```

Comment:
Why do we need this include?

Reply:
okay, we need just this header PortentaEthernet.h, i'll change it